test/fixtures/cookbooks/languages_ruby/recipes/default.rb

```diff
@@ -0,0 +1,8 @@
+include_recipe 'chef-sugar::default'
+include_recipe 'languages::default'
+
+ruby_install '2.1.7'
+
+ruby_install '2.1.7' do
+  prefix '/usr/local/my_ruby'
+end
```

Comment:
I'm not sure why we would _need_ to, the are getting installed in different places. But that's fine.